frontend/catalyst/jit.py

```diff
@@ -660,9 +660,10 @@ def capture(self, args, **kwargs):
         full_sig = merge_static_args(dynamic_sig, args, static_argnums)
 
         if self.compile_options.experimental_capture:
-            return trace_from_pennylane(
-                self.user_function, static_argnums, abstracted_axes, full_sig, kwargs
-            )
+            with EvaluationContext(EvaluationMode.CLASSICAL_COMPILATION):
+                return trace_from_pennylane(
+                    self.user_function, static_argnums, abstracted_axes, full_sig, kwargs
+                )
```

Comment:
I wasn't sure if it makes sense to use the `EvaulationContext` here or inside `trace_from_pennylane`. We already import `EvaluationContext` in `jit.py`, so I chose to use the context manager here. If anyone thinks it makes more sense to use it inside `trace_with_pennylane` to isolate the context further, let me know. It should be a very quick change.

Reply:
No big deal! I think in our tracing pipeline we instantiate the contexts outside of this file, to try to separate modules a bit better.

Reply:
Moved it out of `jit.py`.